lib/project_types/node/messages/messages.yml

```diff
@@ -0,0 +1,5 @@
+node:
```

Comment:
Maybe we could name this `en.yml` in case we ever want to extend to support other languages?

Reply:
That is where I imagined these files going in the future, which is why I put them inside their own folder from the get-go. However, I was worried that naming it with a locale would lead people to think that we already support i18n. What do you think?

Reply:
I don't think that would a huge concern. e.g. when creating a new Rails app, it defaults to have an `en.yml` file (or presumably whatever your machine's locale is set to).

https://github.com/andyw8/rails_6_example/tree/master/config/locales

Reply:
Yeah, I see your point.